g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/translation/service/TranslationSyncServerService.java

```diff
@@ -24,4 +25,6 @@
      * @throws JsonProcessingException 
      */
     public List<TranslationDTO> updateBatchTranslation(List<ComponentMessagesDTO> componentMessagesDTOList) throws L10nAPIException, JsonProcessingException;
+
+    void saveCreationInfo(UpdateTranslationDTO updateTranslationDTO);
```

Comment:
Recommend add "public" key word on method in this interface

Reply:
done

Reply:
Can't change because test coverage will be less than 60%.